skyvern/forge/sdk/routes/agent_protocol.py

```diff
@@ -871,3 +873,54 @@ async def upload_file(
         status_code=200,
         media_type="application/json",
     )
+
+@base_router.get("/tasks/{task_id}/har", tags=["agent"])
+@base_router.get("/tasks/{task_id}/har/", tags=["agent"], include_in_schema=False)
+async def get_task_har(
```

Comment:
Happy to add this -- but you can also use the get_artifact endpoint to get this file!

https://github.com/Skyvern-AI/skyvern/blob/ef95dc6eca7d1ffd7a8744e087f4e4c45e583148/skyvern/forge/sdk/routes/agent_protocol.py#L457

Unless -- the problem is that the file itself is inside the docker image so you can't get it out?

Reply:
Thank you for getting back to us. Retrieving the artifact for a specific step works as well, but please correct me if I'm mistaken. 

I'm unsure if the **get_agent_task_step_artifacts** function can return all the HAR files generated for that specific task or just return the HAR file for one of the task steps.

I'm being impatient here, so I write this function to download all the HAR files at once for a particular task.

Reply:
And yes, I can't get it out because the file is inside the docker image!